diesel.toml

```diff
@@ -3,3 +3,6 @@
 
 [print_schema]
 file = "src/schema.rs"
+with_docs = true
+import_types = ["diesel::sql_types::*"]
```

Comment:
Not sure why/whether we need this. Isn't it working fine without this?

Reply:
we need import types because I may want to use crates like diesel full search. 
So I can do this
import_types = ["diesel::sql_types::*", "diesel_full_text_search::types::*"]